include/glow/Graph/Context.h

```diff
@@ -42,6 +45,9 @@ class Context final {
   /// Maps Placeholders to Tensors.
   PlaceholderMap map_;
 
+  /// Stores trace events for this run. May be nullptr.
```

Comment:
Why is the tracer a part of the context? What is the motivation here?

Reply:
The Context is the per-run part of the state, and traces are also per run. Where else would you put it?